src/core/absorption_chiller.jl

```diff
@@ -36,6 +37,7 @@ Base.@kwdef mutable struct AbsorptionChiller <: AbstractThermalTech
     om_cost_per_ton::Union{Float64, Nothing} = nothing
     macrs_option_years::Float64 = 0
     macrs_bonus_fraction::Float64 = 0
+    heating_load_input::Union{String, Nothing} = nothing
```

Comment:
Because the AbsorptionChiller adds to a heating load, this specifies which heating load is used as input to the AbsorptionChiller.  It's currently set to a single heating load but we can make it flexible with a slight change.

Reply:
Ok, I think that makes sense that the thermal to AbsorpChl only adds to one type of load.